### PR TITLE
Use a tab selector for .NET Standard to help with ease of use

### LINKED ISSUE #27833: Expand the table to make it obvious which netstandard is implement in .net5/6
It kind of stays up for interpretation which versions are compatible/implemented in .NET5/6

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9eb8b774-c6f0-8235-6964-15219baf2ea7
* Version Independent ID: 6819a4a7-7527-d8c7-e132-767f7344e00e
* Content: [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
* Content Source: [docs/standard/net-standard.md](https://github.com/dotnet/docs/blob/main/docs/standard/net-standard.md)
* Product: **dotnet**
* Technology: **dotnet-standard**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

✔️ [Preview: .NET Standard - Select .NET Standard version](https://review.docs.microsoft.com/en-us/dotnet/standard/net-standard?branch=pr-en-us-27837)

Fixes #27833

### CODE REVIEW

File: docfx.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
         "exclude": [
           "samples/**/*.*",
           "**/snippets/**",
-          "**/includes/**",
```
Comment (Youssef1313) [inline docfx.json:20]: I think the removal of `includes` will publish un-intended files.

File: docfx.json [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
-          "samples/**/*.*",
-          "**/snippets/**",
-          "**/includes/**",
-          "**/util/**",
-          "***/contributing.md"
+            "samples/**/*.*",
+            "**/snippets/**",
+            "**/includes/**",
+            "**/util/**",
+            "***/contributing.md"
```
Comment (Youssef1313) [inline docfx.json:22]: nit: Revert indentation to be consistent with the rest of the file.

```suggestion
          "samples/**/*.*",
          "**/snippets/**",
          "**/includes/**",
          "**/util/**",
          "***/contributing.md"
```

File: docfx.json [hunk 3]
```diff
@@ -192,7 +192,8 @@
         "docs/standard/design-guidelines/*.md": "dotnet",
         "docs/standard/security/*.md": "dotnet",
         "docs/visual-basic/**/**.md": "dotnet-visualbasic",
-        "docs/spark/**/**.md": "dotnet-spark"
+        "docs/spark/**/**.md": "dotnet-spark",
+        "includes/**/**.d": "dotnet"
```
Comment (Youssef1313) [inline docfx.json:196]: Typo?

```suggestion
        "includes/**/**.md": "dotnet"
```

And does it actually have an effect?

File: docfx.json [hunk 4]
```diff
@@ -342,7 +344,8 @@
         "docs/standard/threading/**/**.md": "billwagner",
         "docs/standard/whats-new/**/**.md": "gewarren",
         "docs/visual-basic/**/**.md": "KathleenDollard",
-        "docs/whats-new/**/**.md": "billwagner"
+        "docs/whats-new/**/**.md": "billwagner",
+        "includes/**/**.md": "docs"
```
Comment (Youssef1313) [inline docfx.json:348]: Does this have any effect?

Comment (gewarren) [inline docfx.json:348]: Does this resolve to `@dotnet/docs`?

File: includes/net-standard-1.0.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+| .NET and .NET Core          | 1.0, 1.1, 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0                  |
+| .NET Framework <sup>1</sup> | 4.5, 4.5.1, 4.5.2, 4.6, 4.6.1, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8 |
+| Mono                        | 4.6, 5.4, 6.4                                                |
+| Xamarin.iOS                 | 10.0, 10.14, 112.16                                          |
+| Xamarin.Mac                 | 3.0, 3.8, 5.16                                               |
+| Xamarin.Android             | 7.0, 8.0, 10.0                                               |
+| Universal Windows Platform  | 8.0, 8.1, 10.0, 10.0.16299, TBD                              |
+| Unity                       | 2018.1                                                       |
+
+<sup>1</sup> The versions listed for .NET Framework apply to .NET Core 2.0 SDK and later versions of the tooling. Older versions used a different mapping for .NET Standard 1.5 and higher. You can [download tooling for .NET Core tools for Visual Studio 2015](https://github.com/dotnet/core/blob/main/release-notes/download-archives) if you cannot upgrade to Visual Studio 2017 or a later version.
```
Comment (gewarren) [inline includes/net-standard-1.0.md:14]: Does this apply here since it's for .NET Standard 1.0?

File: includes/net-standard-1.1.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+| .NET and .NET Core          | 1.0, 1.1, 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0                  |
+| .NET Framework <sup>1</sup> | 4.5, 4.5.1, 4.5.2, 4.6, 4.6.1, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8 |
+| Mono                        | 4.6, 5.4, 6.4                                                |
+| Xamarin.iOS                 | 10.0, 10.14, 112.16                                          |
+| Xamarin.Mac                 | 3.0, 3.8, 5.16                                                          |
+| Xamarin.Android             | 7.0, 8.0, 10.0                                               |
+| Universal Windows Platform  | 8.0, 8.1, 10.0, 10.0.16299, TBD                              |
+| Unity                       | 2018.1                                                       |
+
+<sup>1</sup> The versions listed for .NET Framework apply to .NET Core 2.0 SDK and later versions of the tooling. Older versions used a different mapping for .NET Standard 1.5 and higher. You can [download tooling for .NET Core tools for Visual Studio 2015](https://github.com/dotnet/core/blob/main/release-notes/download-archives) if you cannot upgrade to Visual Studio 2017 or a later version.
```
Comment (gewarren) [inline includes/net-standard-1.1.md:14]: Same here and for other < 1.5 tables.

File: docs/standard/net-standard.md [hunk 7]
```diff
@@ -13,20 +13,8 @@ ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 
 For information about choosing between .NET 5+ and .NET Standard, see [.NET 5+ and .NET Standard](#net-5-and-net-standard) later in this article.
 
-## .NET implementation support
-
-The various .NET implementations target specific versions of .NET Standard. Each .NET implementation version advertises the highest .NET Standard version it supports, a statement that means it also supports previous versions. For example, .NET Framework 4.6 implements .NET Standard 1.3, which means that it exposes all APIs defined in .NET Standard versions 1.0 through 1.3. Similarly, .NET Framework 4.6.1 implements .NET Standard 1.4, while .NET 5 implements .NET Standard 2.1.
-
-The following table lists the **minimum** implementation versions that support each .NET Standard version. That means that later versions of a listed implementation also support the corresponding .NET Standard version. For example, .NET Core 2.1 and later versions support .NET Standard 2.0 and earlier versions.
-
-[!INCLUDE [net-standard-table](../../includes/net-standard-table.md)]
```
Comment (gewarren) [inline docs/standard/net-standard.md:22]: We should probably delete this include file as part of this PR. Or is it used elsewhere?

File: includes/net-standard-1.0.md (hunk 5 above)
Comment (IEvangelist) [reply includes/net-standard-1.0.md:14]: Yes, I believe so. This is the .NET Standard 1.0 file.

File: includes/net-standard-1.1.md (hunk 6 above)
Comment (IEvangelist) [reply includes/net-standard-1.1.md:14]: I believe this is applicable, but I'm curious - why wouldn't it be? I might be missing something.

File: docs/standard/net-standard.md (hunk 7 above)
Comment (IEvangelist) [reply docs/standard/net-standard.md:22]: Done

File: docfx.json (hunk 4 above)
Comment (IEvangelist) [reply docfx.json:348]: Yes, I believe so.

File: docfx.json [hunk 8]
```diff
@@ -192,7 +192,8 @@
         "docs/standard/design-guidelines/*.md": "dotnet",
         "docs/standard/security/*.md": "dotnet",
         "docs/visual-basic/**/**.md": "dotnet-visualbasic",
-        "docs/spark/**/**.md": "dotnet-spark"
+        "docs/spark/**/**.md": "dotnet-spark",
+        "includes/**/**.md": "dotnet"
```
Comment (BillWagner) [inline docfx.json:196]: Does this have any effect?

File: includes/net-standard-1.0.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+.NET Standard 1.0 has 7,949 of the 37,118 available APIs.
+
+| .NET implementation         | Version support                                              |
+|-----------------------------|--------------------------------------------------------------|
```
Comment (BillWagner) [inline includes/net-standard-1.0.md:4]: Should we remove the non-LTS versions that are out of support here? Or is this meant to be a historically accurate statement?

**Review by BillWagner — COMMENTED** [review]
I like this @IEvangelist 

I had a few questions. Once we resolve those (and other comments), this is ready.

File: includes/net-standard-1.0.md (hunk 9 above)
Comment (IEvangelist) [reply includes/net-standard-1.0.md:4]: This is meant to highlight all possible implementations of .NET Standard versions. It's a map of what .NET Implementation aligns with a given .NET Standard version.

File: docfx.json (hunk 8 above)
Comment (IEvangelist) [reply docfx.json:196]: Removing this.

File: includes/net-standard-1.1.md (hunk 6 above)
Comment (gewarren) [reply includes/net-standard-1.1.md:14]: Because the different mapping only applies to .NET Standard 1.5 and higher.

Comment (IEvangelist) [reply includes/net-standard-1.1.md:14]: I'll remove this for refs below 1.5, thanks

**Comment by IEvangelist** [comment]
@gewarren - this one should be good-to-go now, thanks.

**Comment by gewarren** [comment]
@IEvangelist Can you also delete net-standard-table.md?

**Comment by IEvangelist** [comment]
> @IEvangelist Can you also delete net-standard-table.md?

@gewarren done

**Review by gewarren — APPROVED** [review]

**Comment by cremor** [comment]
@IEvangelist This PR broke the list of supported .NET and .NET Core versions for .NET Standard 2.1. It now claims that .NET Standard 2.1 is even supported by .NET Core 1.0.

I didn't check everything else, but that mistake was appearent. Maybe check all tables again?

**Comment by IEvangelist** [comment]
> @IEvangelist This PR broke the list of supported runtime versions for .NET Standard 2.1. For example, it now claims that .NET Standard 2.1 is even supported by .NET Core 1.0.
> 
> Maybe check all tables again against the old doc? Or against https://dotnet.microsoft.com/en-us/platform/dotnet-standard#versions

Thank you @cremor, great catch. Fixed here: #28076

**Pull request merged**